scripts/items/tidal_talisman.lua

```diff
@@ -43,7 +21,7 @@ itemObject.onItemCheck = function(target)
 end
 
 itemObject.onItemUse = function(target)
-    target:setPos(unpack(teleportData[target:getZoneID()]))
+    target:addStatusEffectEx(xi.effect.TELEPORT, 0, xi.teleport.id.TIDAL_TALISMAN, 0, 2)
```

Comment:
missed a 2. soon as thats updated this pr is ready to merge.

Reply:
-10 to mowfordor